param/parameterized.py

```diff
@@ -1490,19 +1490,24 @@ def trigger(self_, *param_names):
         will be triggered whether or not the parameter values have
         actually changed.
```

Comment:
```suggestion
        actually changed. As a special case, the value will actually be changed for a Parameter
        of type Trigger, setting it to true so that it is clear which Trigger parameter has been
        triggered.
```

Reply:
Used this as the template for the docstring pushed in af0a5d8